### PR TITLE
Text generator

### PR DESCRIPTION
**Pull request by Kaylotura**
Generates a list of two strings using a simple quick and dirty sample.txt file.
There is a Class Module for Markov Chains, and a Run Module that creates the list.
Occasionally the text generator will fail. Working on a fix for this, and cleaning up basic pep8 style.

### CODE REVIEW

File: markov_chain.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+"""Generates a pseudo-random string of text using a markov chain language model and a text file source ."""
```
Comment (davidselassie) [inline markov_chain.py]: If you sourced this file from another project and are just including it directly, please include a comment with a URL of its source.

File: markov_chain.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        """Generates a phrase from the sample text anywhere between 1 and 10 tokens long."""
+        seed = random.randint(0, self.word_size - 3)
+        seed_word, next_word = self.words[seed], self.words[seed + 1]
+        w1, w2 = seed_word, next_word
+        gen_words = []
+        for i in range(size):
+            gen_words.append(w1)
+            w1, w2 = w2, random.choice(self.cache[(w1, w2)])
+        gen_words.append(w2)
+        return ' '.join(gen_words)
```
Comment (davidselassie) [inline markov_chain.py]: Both of these functions are identical except for the default argument for `size`. I would not duplicate this code and instead in your `text_generator` module, have the calling functions set those parameters. Then this module just deals with Markov in the abstract and your text generator module is specifically for memes.

File: text_generator.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+def get_top_line(markov):
+    """Generates a single string using Markov chains of 1 to 6 tokens"""
+    return (markov.generate_top_text()).upper()
+
+
+def get_bottom_line(markov):
+    """Generates a single string using Markov chains of 1 to 11 tokens"""
+    return (markov.generate_bottom_text()).upper()
```
Comment (davidselassie) [inline text_generator.py]: These two functions make a great interface for the rest of the program.

**Comment by davidselassie** [comment]
Ship it! If other people like this PR, they should also reply with Ship It! and then the person who makes it can merge it into master.

**Comment by coreyadkins** [comment]
Looks good to me! Ship it!

**Pull request merged**